front/app/components/UI/UserSelect/index.tsx

```diff
@@ -95,7 +95,6 @@ const UserSelect = ({
     } else if (option.attributes) {
       return (
         <UserOption>
-          {!hideAvatar && <Avatar user={option} />}
```

Comment:
Removed the Avatar. There was a bug in it that I couldn't easily figure out. I was comparing it to other work we could do and this was not valuable here. Also, it takes up quite some space in the user selector, so I think it's better without the Avatar.

Reply:
(Also, if we would want the avatar back, we should rather use our existing Avatar component)